### DISCUSSION TITLE
zh/note/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#536] [Announcements]
# zh/note/

(≧∇≦)ﾉ 今朝のブログへよこそう!ヾ(≧▽≦*)o 目的 以前遇到好的技术博客，开源书籍或教程，只是浏览一遍或者就直接丢进收藏夹再也没看过了（收藏从未停止，学习从未开始~） 现在将自己学习的过程记录下来，只有脚踏实地去实践一遍才知道自己理解是否到位，也能增加自己的熟悉程度 Talk is cheap, show me the code ！

http://localhost:8080/zh/note/

**Comment by dreamjz** [comment]
Just test (≧∇≦)ﾉ
